# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../esmart/libdir.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- esmart/configure.in_orig	2007-02-15 00:03:11.000000000 +0100
+++ esmart/configure.in	2007-02-15 00:04:03.000000000 +0100
@@ -27,7 +27,7 @@
   AC_DEFINE_UNQUOTED(PREFIX, "${ac_default_prefix}", [Installation Prefix])
 else
   AC_DEFINE_UNQUOTED(PREFIX, "${prefix}", [Installation Prefix])
-  container_plugin_dir=${prefix}/lib/${PACKAGE}/layout
+  container_plugin_dir=${libdir}/${PACKAGE}/layout
 fi
 
 AC_DEFINE_UNQUOTED(CONTAINER_PLUGIN_DIR, "${container_plugin_dir}", [Container Plugin Directory])
